docs/data-modeling/editing.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Editable table data
+summary: Admins can create, update, and delete data in tables right in Metabase.
+---
+
+# Editable table data
```

Comment:
Editing table data? Editable table? Table editing? Editing data in tables? idk what's better, but "Editable table data" is very hard to parse, we should come up with something better